tjeaneric/Flashcards-Frontend
# Finished page for all links table

#### What does this PR do?
- Authenticated users can view all cards page
#### Description of Task to be completed?
- Authenticated users can view the all cards page
#### How should this be manually tested?
- * clone the repo using SSH: `git clone git@github.com:tjeaneric/Flashcards-Frontend.git`
                          or    HTTPS: `git clone https://github.com/tjeaneric/Flashcards-Frontend.git`
- run `git checkout ft-all-cards-page`
- then `npm install`
- run `npm run dev`
- then check the route for creating a card on `localhost:{port}/all-cards`
- test if you can access all cards page when authenticated
#### Screenshots
![all-cards-flashcards](https://user-images.githubusercontent.com/81187224/181918286-6e1379c3-54c7-4593-8bda-b1b964ebbd03.png)
